openedx_proversity_reports/reports/enrollment_report.py

```diff
@@ -0,0 +1,82 @@
+"""
+Enrollment report Report Class.
+"""
+import logging
+from datetime import datetime
+
+from opaque_keys import InvalidKeyError
+from opaque_keys.edx.keys import CourseKey
+
+from openedx_proversity_reports.edxapp_wrapper.get_student_account_library import get_user_salesforce_contact_id
+from openedx_proversity_reports.edxapp_wrapper.get_student_library import get_course_enrollment, get_user_profile
+
+
```

Comment:
Is this space necessary?, is there a Python convention to separate these kind of statements? @andrey-canon